extensions/hibernate-search-backend-elasticsearch/deployment/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-hibernate-search-backend-elasticsearch-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-hibernate-search-backend-elasticsearch-deployment</artifactId>
+    <name>Quarkus - Hibernate Search - Elasticsearch - Deployment</name>
```

Comment:
I have only one comment for this PR: for common/internal extensions that are not to be consumed, we usually have a `-common` or `-internal` suffix to the artifactId.

I actually use it in the config doc generation (using unlisted is unfortunately not the semantic I need). I think it would be worth adding the `-common` suffix here.

Reply:
Thanks for the review.

> I think it would be worth adding the -common suffix here.

I think it would be weird since there is no other artifact starting with `quarkus-hibernate-search-backend-elasticsearch`... So there's no obvious answer to the question "common to what?".

But if the tooling requires it, so be it.
Users aren't supposed to reference this artifact directly anyway, so the weirdness shouldn't reach them.